client/src/components/Address/Address.js

```diff
@@ -23,25 +24,170 @@ const Address = () => {
   let decoded = parseJwt(getJWT());
   let [user_id, setUser_id] = useState(decoded.id);
   const navigate = useNavigate();
+  let [razor, setRazor] = useState(null);
+  let [name, setName] = useState(null);
+  let [email, setEmail] = useState(null);
+  let [phone, setPhone] = useState(null);
+
+  const nullMessage = () => {
+    return setTimeout(() => {
+      setMessage(null);
+    }, 6000);
+  };
 
   useEffect(() => {
+    axios
+      .post(`/user/checkout/payment/${user_id}`)
+      .then((res) => {
+        setRazor(res.data);
+      })
+      .catch((err) => {
+        setMessage("error in razor pay");
+        nullMessage();
+      });
+
     axios
       .get(`/user/address/get/${user_id}`)
       .then((res) => {
         if (res.data.length == 0) {
           setMessage("No address found!! Please add new one");
+          nullMessage();
         } else {
           setAddress(res.data);
           setAddress_id(res.data[0].id);
+          try {
+            getById(res.data[0].id);
+          } catch (error) {
+            setMessage("error in getting address");
+            nullMessage();
+          }
           document.getElementById(res.data[0].id).style.border =
             "solid 2px blue";
         }
       })
       .catch((err) => {
         setMessage("Sorry! Something went wrong. Please Try again", err);
+        nullMessage();
       });
   }, []);
+  const displayRazorPay = async () => {
```

Comment:
can you simplify this function? refactor code

Reply:
i did it.